sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/BoundAttribute.scala

```diff
@@ -84,17 +84,27 @@ object BindReferences extends Logging {
       expression: A,
       input: Seq[Attribute],
```

Comment:
Actually, yeah: in `GenerateMutableProjection` we use the same InputSchema for every expression.